include/net/net_ip.h

```diff
@@ -902,7 +902,7 @@ int net_addr_pton(sa_family_t family, const char *src, void *dst);
  * @param family IP address family (AF_INET or AF_INET6)
  * @param src Pointer to struct in_addr if family is AF_INET or
  *        pointer to struct in6_addr if family is AF_INET6
- * @param dst IP address in a non-null terminated string
+ * @param dst Buffer for IP address as a null terminated string
```

Comment:
...or "Pointer to" as we have for src param?

Reply:
Well, that param is not just pointer, it's buffer. Well, it's pointer to buffer, but that's too long, and everyone familiar with C knows that buffers are passed by pointers.

Also, next param's docstr say: "size Number of bytes available in the buffer". Buffer, buffer, which buffer? Now dst's docstr explains which.

Reply:
"Pointer to a buffer" aligns with how "src" param is described. But I am also fine with the current proposal :)